collections/_posts/2021-07-09-how-to-pick-a-chia-pool.en.md

```diff
@@ -0,0 +1,70 @@
+
+Related to experience is a pool operator’s reputation. Their reputation does not have to be in the pooling provider business; it can be from a number of other projects and businesses they have worked on. You have to be the judge here as to what is relevant to you in order to decide if a pooling provider will be responsible for paying out your fair share and providing the security while doing so!
+
+As an example of balancing personal thresholds of reputation and trust, let's look at an example that many experienced crypto users have heard about: Nicehash. Nicehash is a big provider for pooling several cryptos together. They were once victims of an attack that cost them and their users millions of dollars worth of cryptocurrency. However, through good transparency, communication, and working hard to mitigate and recover from the effects of a very bad situation, they managed to be around still today. While there is an occasional mixed debate on if their efforts were thorough enough, they clearly came out of the situation strongly, taking actions that instilled confidence in enough of their users with a successful pooling operation still in place. While every farmer has a different perspective and personal threshold on what is a good reputation or level of trust, knowing and understanding what your own threshold for what good looks like is key.
+
+Community engagement is also important for many people. This doesn’t necessarily mean engagement on a day-to-day level, but should you have problems, it’s good to have an easy way to get in touch with someone that can provide some kind of direction to support you in your time of need. Some providers are regularly available through services like Telegram or Discord or even through Facebook or Twitter.
+
+## Transparency and communication
+
+The transparency that a pool provides is another important factor. First and foremost is their payment structure for individual farmers' shares of the profits of pooled farming. How are they going to reward you for your part of the pool? It is very common for pools to operate on a percentage of each farmer's earnings. Running a pool isn't an easy task and takes considerable resources and technical knowledge to do it at a basic satisfactory level. For their work, it is only fair that a pool operator will charge you, the farmer, for providing the service that they do. Many farms have settled over time into a standard of setting fees in the 1-3% range of the rewards. Another thing to be aware of is how they structure their shares, when and how you are earning shares, and when those shares are no longer counted. 
+
+Also, consider what the process of leaving their pool is like–how long does it take to "free" your plots from being locked to the pool, and what happens to any shares that you have active?
```

Comment:
We need to make sure that the leave time is not set by us and not the pools.

Reply:
What about rewording to something like:

Also, it's good to consider what the process of leaving their pool is like. The system allows pools to establish their own "cool-down" timer for how long between when you initiate a pool-leave to when your plots are finally free from it. This can be as short as a few minutes or as long as several hours, at the pools own discretion (though a generally accepted industry average is 30 minutes). A pool with a long enough time to prevent cheaters from taking shares without contributing but not so long as to lock you up in their pool needlessly long after you have chosen to leave is one that is looking out for both you and the pool fairly.